lib/components/explorer_results/explorer_result_page_transaction_item.dart

```diff
@@ -143,35 +104,35 @@ class ExplorerResultPageTransactionItem extends StatelessWidget {
                   children: [
                 Text(l10n.transactionItemLabelTransactionId,
                     style: itemHeaderType(context)),
-                Text(transaction.id),
-              ])),
-          CopyButton(copiedText: transaction.id),
-        ]),
-        ThemedControls.spacerVerticalSmall(),
-        Flex(direction: Axis.horizontal, children: [
-          Expanded(
-              child: Column(
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  children: [
-                Text(l10n.transactionItemLabelTransactionDigest,
-                    style: itemHeaderType(context)),
-                Text(transaction.digest),
+                Text(transaction.transaction.txId.toString()),
               ])),
-          CopyButton(copiedText: transaction.digest),
+          CopyButton(copiedText: transaction.transaction.txId.toString()),
         ]),
+        // ThemedControls.spacerVerticalSmall(),
+        // Flex(direction: Axis.horizontal, children: [
```

Comment:
do we need to keep this code?

Reply:
Of course not, I will make sure to remove any commented code in the coming commits.